modules/internal/String.chpl

```diff
@@ -1567,6 +1567,7 @@ module String {
             }
             offset += sLen;
           }
+          joined.buff[offset] = 0;
```

Comment:
Wouldn't it make more sense for this line to be after the loop?

Reply:
Thanks!  I missed that because the following brace was mis-indented.

Reply:
Oh, just my copy was mis-indented.  That's odd.  In any case thanks for catching it.